.env.example

```
@@ -4,4 +4,4 @@ OPENAI_CHAT_MODEL_ID="qwen3-vl-30b-a3b-instruct"
 OPENAI_RESPONSES_MODEL_ID="qwen3-vl-30b-a3b-instruct"
 OPENAI_API_KEY="lm-studio"
 CHANAKYA_DEBUG=true
-DATABASE_URL="sqlite:////home/rishabh/github_projects/MAF-demo/chanakya_data/chanakya.db"
\ No newline at end of file
+DATABASE_URL="sqlite:////home/rishabh/github_projects/MAF-demo/chanakya_data/chanakya.db"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)